apps/docs/src/basics/iconography.md

```diff
@@ -1,3 +1,68 @@
 # Iconography
 
+:::warning Attention
+Check out the whole icon palette in the [resources](/resources/icons) documentation.
+Feel free to browse through all the assets there.
+:::
+
+Our system icons are designed to be simple with a touch of detail. They are characteristic yet easy to understand.
+
+![Image](/assets/icons-overview.png)
```

Comment:
The "Image" alt text is used for screen readers so its best to set it to something meaningful for each image that describes its content